plugins/lua/stockflow.lua

```diff
@@ -527,6 +530,14 @@ function collect_reactions()
                     {job_types.MakeFlask, "Forge", "Flask"},
                     {job_types.MakeChain, "Forge", "Chain"},
                     {job_types.MakeCrafts, "Make", "Crafts"},
+                    {job_types.MakeFigurine, "Make", "Figurine"},
+                    {job_types.MakeAmulet, "Make", "Amulet"},
+                    {job_types.MakeScepter, "Make", "Scepter"},
+                    {job_types.MakeCrown, "Make", "Crown"},
+                    {job_types.MakeRing, "Make", "Ring"},
+                    {job_types.MakeEarring, "Make", "Earring"},
+                    {job_types.MakeBracelet, "Make", "Bracelet"},
+                    {job_types.MakeGem, "Make Large", "Gem"},
```

Comment:
The split is where the material's name goes.  In the manager, the line is "Make Large iron Gem" or "Make Large wooden Gem" and I'm replicating that here.  Technically, the "Large" corresponds to the adjective field in itemdefs, as seen on trap components and large pots, but crafts seem to be hard-coded.